docs/ui_configuration.md

```diff
@@ -0,0 +1,88 @@
+# Configuring the UI
+
+The admin has the ability to configure most parts of the UI to some extent.
+
+The Ui configuration section has this structure by default:
+
+```
+ui:
+    gpuConfig:
+        type: dropdown
+        config:
+            start: 1
+            end: 10
+    imageConfig:
+        blacklist: [unavailable_image1:v0.1, unavailable_image2:v0.2]
+        sort: 'name'
+    sizeConfig:
+        enabled: false
+    envVarConfig:
+        freq_keys: [frequent_variable1, frequent_variable2]
```

Comment:
I think this needs to take into account a possiblity that the envvar is a secret

```
frequent_keys:
  - name: f_var1
    type: text
  - name:f_var2
    type: password
```

`type` can be omitted and default it `text`